packages/compiler/core/util.ts

```diff
@@ -419,8 +419,8 @@ export function getSourceFileKindFromExt(path: string): SourceFileKind | undefin
   const ext = getAnyExtensionFromPath(path);
   if (ext === ".js" || ext === ".mjs") {
     return "js";
-  } else if (ext === ".cadl") {
-    return "cadl";
+  } else if (ext === ".tsp") {
```

Comment:
We are supporting both extensions with a follow-on PR right?

Reply:
https://github.com/Azure/cadl-azure/issues/2597